mcq/views/finish_mcq_test.py

```diff
@@ -61,6 +61,58 @@
                                 "hasUserAnswered": False
                             }
                         ],
+                        "unknownWords": [
```

Comment:
if you increase/decrease the confidence of the unknown words in the finish_mcq_test controller, are these reflected in the response

Reply:
Mobile does not use the response, but yes.